tests/gnrc_dhcpv6_client/Makefile

```diff
@@ -41,6 +41,8 @@ include $(RIOTBASE)/Makefile.include
 
 .PHONY: dhcpv6_server
 
+test-with-config/check-config: IFACE := ${IFACE}
+
 dhcpv6_server: IFACE := ${IFACE}
```

Comment:
```suggestion
dhcpv6_server: export IFACE := ${IFACE}
```

Reply:
Same here you can use a common `$(call target-export-variables,$(RULES_THAT_NEED_THIS_EXPORT),IFACE)`, its basically doing the same as `export IFACE:= ${IFACE}`, but you can so it all in one place

Reply:
so then it comes down to whether 
```
dhcpv6_server: export IFACE := ${IFACE}

test-with-config/check-config: export IFACE := ${IFACE}
```

or
```
EXPORT_IFACE_FOR = dhcpv6_server test-with-config/check-config
$(call target-export-variables,$(EXPORT_FACE_FOR),IFACE)
```
is more readable

Reply:
Done as proposed by @fjmolinas